web/src/repo/backend.ts

```diff
@@ -101,28 +110,28 @@ export const resolveRev = memoizeObservable(
                 if (!data) {
                     throw createAggregateError(errors)
                 }
-                if (data.repository && data.repository.redirectURL) {
-                    throw new RepoSeeOtherError(data.repository.redirectURL)
-                }
-                if (!data.repository) {
+                if (!data.repositoryRedirect) {
                     throw new RepoNotFoundError(ctx.repoName)
                 }
-                if (data.repository.mirrorInfo.cloneInProgress) {
-                    throw new CloneInProgressError(ctx.repoName, data.repository.mirrorInfo.cloneProgress || undefined)
+                if (data.repositoryRedirect?.__typename === 'Redirect') {
```

Comment:
You already checked for the presence above :)
```suggestion
                if (data.repositoryRedirect.__typename === 'Redirect') {
```